data/core.yaml

```diff
@@ -1113,6 +1113,9 @@ en:
     msBuildings:
       label: Microsoft Buildings
       license_markdown: "[license](https://github.com/microsoft/USBuildingFootprints/blob/master/LICENCE-DATA)"
+    rapidMapillaryFeatures:
```

Comment:
This is where all the string resources are kept. Code can reference these features with dot-separated string keys, hierarchically.  

So that means that the string 'label' on line 1117 would be referenced from code using `context.t('rapid_feature_toggle.rapidMapillaryFeatures.label')`